.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "eslint.workingDirectories": ["src"]
+}
```

Comment:
I'm always a little hesitant about committing editor-specific configurations. By itself, this is actually worthwhile, but I worry about what other cruft might get added to this file.